kura/org.eclipse.kura.wire.component.conditional.provider/src/main/java/org/eclipse/kura/internal/wire/conditional/ConditionalOptions.java

```diff
@@ -18,6 +18,11 @@
 
 import javax.script.ScriptException;
 
+/**
+ * 
+ * @deprecated as of Kura 5.3.0
+ */
+@Deprecated
```

Comment:
Uuuuh actually, there's an extra `*` for each of these type of comments. Feel free to ignore this if you don't want to fix it.

```suggestion
/**
 * @deprecated as of Kura 5.3.0
 */
@Deprecated
```

Reply:
Fixed.